swift/apple/FirezoneNetworkExtension/NetworkSettings.swift

```diff
@@ -38,7 +38,7 @@ class NetworkSettings {
 
     // Set tunnel addresses and routes
     let ipv4Settings = NEIPv4Settings(addresses: [tunnelAddressIPv4!], subnetMasks: ["255.255.255.255"])
-    let ipv6Settings = NEIPv6Settings(addresses: [tunnelAddressIPv6!], networkPrefixLengths: [128])
+    let ipv6Settings = NEIPv6Settings(addresses: [tunnelAddressIPv6!], networkPrefixLengths: [120])
```

Comment:
Ah, yes I remember removing the workaround here that we had from the WireGuard client. I tested it at the time, and seemed GTG, but we did not have full-route. This must be the bug they hit.